index.html

```diff
@@ -11,10 +11,31 @@ <h2>Welcome to this beautiful webpage</h2>
     <img src=https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/23.png />
     <!-- placeholder here to avoid merge conflicts-->
     <!-- placeholder here to avoid merge conflicts-->
-    <p>
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quo, rerum nostrum? Ad deleniti facere in totam
-        officiis? Ea inventore, repudiandae non natus quod illum accusamus maxime ipsum laboriosam impedit. Dolores.
-    </p>
+    <div style="width:375px; margin:25px auto; font-family:Arial">
```

Comment:
(nit, not in scope of the codelab) -- let's aim to use classes to do styling -- e.g. 

index.html
```html
<div class="poem-container">
  blah
</div>
```
css file
```html
.poem-container {
  width: blah;
  margin: 25px auto;
  ....
}
```

Reply:
Totally agree on classes vs inline for "real" code.  I was just being lazy. :)